contents/product-engineers/how-to-do-ab-testing.md

```diff
@@ -0,0 +1,103 @@
+---
+title: How to think about running your first A/B test (why, when, and where)
+date: 2024-01-17
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/experiment-hog.jpg
+featuredImageType: full
+tags: 
+  - AB testing
+  - Product engineers
```

Comment:
```suggestion
title: How, when and where to run your first A/B test
date: 2024-01-26
author: ["ian-vanagas"]
featuredImage: ../images/blog/experiment-hog.jpg
featuredImageType: full
tags: 
  - AB testing
  - Product engineers
seo: {
  metaTitle: "How to do A/B testing – a beginner's guide",
  metaDescription: "When is the right time to do run an A/B test? What should you test and how to do you run A/B tests? This is everything you need to know about running your first A/B test."
} 
```

Not massively keen on "a beginner's guide" in the SEO title, so open to suggestions there.